cqc/pythonLib.py

```diff
@@ -157,7 +168,7 @@ def get_remote_from_directory_or_address(cqcNet, name, remote_socket):
         remote_port = addr[4][1]
     return remote_ip, remote_port
 
-
+# !! Deprecated. Do not use this method
```

Comment:
Change this to a deprecation warning - nobody will find a comment: https://docs.python.org/3.7/library/exceptions.html#DeprecationWarning

Reply:
Also - all deprecation changes must be approved by Axel as lead maintainer

Reply:
Included a deprecation warning